lib/streamlit/util.py

```diff
@@ -130,7 +134,7 @@ def index_(iterable, x) -> int:
     for i, value in enumerate(iterable):
         # https://stackoverflow.com/questions/588004/is-floating-point-math-broken
         # https://github.com/streamlit/streamlit/issues/4663
```

Comment:
nit: not really part of your change, but I'd remove these comments as I don't think either of them are particularly helpful given that needing to take floating point error into account shouldn't be anything surprising

Reply:
Comments removed.